pkg/runner/runner.go

```diff
@@ -542,10 +551,36 @@ func createCommand(details *RunDetails) (string, *exec.Cmd, error) {
 
 func createJavaCommand(javaCmd string, details *RunDetails) (*exec.Cmd, error) {
 	args := details.getCommand()
+	appDir := details.getenvOrDefault(v1.EnvVarCohAppDir, "/app")
+	jibMainClassFileName := appDir + "/jib-main-class-file"
```

Comment:
I think this is in the wrong place. We determine the main class to use in the `func server(details *RunDetails, _ *cobra.Command)` line 43 of `cmd_server.go`. We get the value of the environment variable `v1.EnvVarAppMainClass` which will be set if the user specified a main class in their yaml. If this is not set, i.e. `found == false` then do your stuff before the `switch`. If you find a JIB main class in the file set `mc` to be the class name and set `found = true`.

Reply:
That makes a lot of sense. Thanks JK. I'll make that change